rkt/image/filefetcher.go

```diff
@@ -22,14 +22,14 @@ import (
 
 	"github.com/coreos/rkt/pkg/keystore"
 	rktflag "github.com/coreos/rkt/rkt/flag"
-	"github.com/coreos/rkt/store"
+	"github.com/coreos/rkt/store/acistore"
 	"github.com/hashicorp/errwrap"
 )
 
 // fileFetcher is used to fetch files from a local filesystem
 type fileFetcher struct {
 	InsecureFlags *rktflag.SecFlags
-	S             *store.Store
+	S             *acistore.ACIStore
```

Comment:
Hmm, while we're here, `fileFetcher` is private, but fields are public. Does this make sense?